publish_wc.py

```diff
@@ -0,0 +1,166 @@
+
+#!/usr/bin/python
+
+# sucscribes to log_ranges and publishes PointStanmed message in LC
+# if transformation between 'drone' and 'world' exists, publishes PointStamped message in WC
+# publishes laserscan in 'drone' frame, and converts it to PointCloud2 too.
```

Comment:
This needs to be a doc-string
```suggestion
"""
sucscribes to log_ranges and publishes PointStanmed message in LC
if transformation between 'drone' and 'world' exists, publishes PointStamped message in WC
publishes laserscan in 'drone' frame, and converts it to PointCloud2 too.
"""
```